README.md

```diff
@@ -50,6 +50,7 @@ Ensure the properties there fit your environment.
 - Without tests: `mvn clean package -DskipTests `
 - With unit tests: `mvn clean package`
 - With integration tests: `mvn clean package -Pintegration`
+- With component preload file generated (requires npm):  `mvn clean package -PgenerateComponentPreload`
```

Comment:
Please add NPM to [build requirements](https://github.com/AbsaOSS/enceladus#build-requirements) as optional requirement

Reply:
Also needs grunt.

Reply:
That is actually an issue. Grunt should be pulled automatically by the build process. Jan si looking into it.

Reply:
There is a [frontend-maven-plugin](https://github.com/eirslett/frontend-maven-plugin) which can automatically install these for you as part of the maven build process. Not sure if it's something you're interested in. Just putting it out there.

Reply:
Yeah used that in the latest update. Seems to work smooth.

So in the latest version, no dependencies on npm or node, all installed automatically with the profile.